upup/pkg/fi/cloudup/bootstrapchannelbuilder/bootstrapchannelbuilder.go

```diff
@@ -557,6 +557,27 @@ func (b *BootstrapChannelBuilder) buildAddons(c *fi.ModelBuilderContext) (*chann
 		}
 	}
 
+	if b.Cluster.Spec.AWSLoadBalancerController != nil && fi.BoolValue(b.Cluster.Spec.AWSLoadBalancerController.Enabled) {
+
+		key := "aws-load-balancer-controller.addons.k8s.io"
+		version := "2.0.0"
```

Comment:
If not a problem, we should fo with latest.
```suggestion
		version := "2.1.2"
```